src/k8s/components/gitserver/configuration/toInternal.dhall

```diff
@@ -26,6 +31,9 @@ let Configuration/ResourceRequirements/toK8s =
 let simple/gitserver =
       (../../../../simple/gitserver/package.dhall).Containers.gitserver
 
+let Optional/default =
```

Comment:
don't know how we should do it, but we should enforce a version. we have 

```
src/k8s/util/functions/list-to-optional.dhal
```

with imports pointing at v20